### PR TITLE
added routing + closing conversations from previous commits

### PR DESCRIPTION
**Pull request by ufoBeliever**
added routing + closing conversations from previous commits

### CODE REVIEW

File: src/components/Typography/types.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Types for typography component
+ * @param {number} size - Size of the text (pixels)
+ * @param {string} label - Text content of the compopnent
+ */
+
+export interface ITypographyProps extends Partial<HTMLHeadingElement> {
+  size: number;
+  label: string;
```
Comment (igorizviekov) [inline src/components/Typography/types.ts:9]: please rename the file name to be `typography.types.d.ts`

File: src/sections/Home/index.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  .test-block__list {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    flex-direction: column;
+    border: 2px solid;
+    padding: 10px;
+    width: 20%;
+
+    .test-block__link {
```
Comment (igorizviekov) [inline src/sections/Home/index.scss:15]: syntax should be refactored to `&__link`, same applies to the rest

File: src/components/Typography/Typography.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { ITypographyProps } from './types';
+
+export const Typography: React.FC<ITypographyProps> = ({ size, label }) => {
```
Comment (igorizviekov) [inline src/components/Typography/Typography.tsx:4]: also pass to h2 the rest of the props from the default heading interface youʼre extending from

**Review by igorizviekov — CHANGES_REQUESTED** [review]

File: src/components/Typography/types.ts (hunk 1 above)
Comment (ufoBeliever) [reply src/components/Typography/types.ts]: https://github.com/igorizviekov/mt-booking_app/pull/20/commits/acfb9043d165fe9a9b30dc621c1b84e1e4d5bf39

File: src/sections/Home/index.scss (hunk 2 above)
Comment (ufoBeliever) [reply src/sections/Home/index.scss]: https://github.com/igorizviekov/mt-booking_app/pull/20/commits/dfe1bada6858a2aeb46a8f40606dd5403a27e98e

File: src/components/Typography/Typography.tsx (hunk 3 above)
Comment (ufoBeliever) [reply src/components/Typography/Typography.tsx]: https://github.com/igorizviekov/mt-booking_app/pull/20/commits/32039294d6534ccbdc05c306ad374bcc95efda94

**Review by igorizviekov — APPROVED** [review]

**Pull request merged**